server/src/test/java/org/elasticsearch/health/node/HealthInfoTests.java

```diff
@@ -74,6 +83,18 @@ public static RepositoriesHealthInfo randomRepoHealthInfo() {
         return new RepositoriesHealthInfo(randomList(5, () -> randomAlphaOfLength(10)), randomList(5, () -> randomAlphaOfLength(10)));
     }
 
+    private static FileSettingsHealthInfo mutateFileSettingsHealthInfo(FileSettingsHealthInfo original) {
+        long changeCount = original.changeCount() ^ randomLongBetween(1, 100); // Always different from original
```

Comment:
Nit:
```suggestion
        long changeCount = randomValueOtherThan(original.changeCount(), ESTestCase::randomLong);
```
this is an optional suggestion, this is just more aligned with what we do in the rest of the codebase and might thus be a bit easier to read. Your approach is not hard to read of course, that's why it's optional.